pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-signald/default.nix

```diff
@@ -19,9 +19,7 @@ stdenv.mkDerivation rec {
   ];
 
   PKG_CONFIG_PURPLE_PLUGINDIR = "${placeholder "out"}/lib/purple-2";
-  PKG_CONFIG_PURPLE_DATADIR = "${placeholder "out"}/share";
-
-  installFlags = [ "DESTDIR=$(out)" ];
+  PKG_CONFIG_PIDGIN_DATADIR = "${placeholder "out"}/share";
```

Comment:
I think it should still use `PKG_CONFIG_PURPLE_DATADIR`, so that Finch and other clients can use it, but the installFlags are probably causing the problem.

Reply:
I've changed this, as this is used within `Makefile`: https://github.com/hoehermann/purple-signald/blob/v0.11.0/Makefile#L28
But I'm not used to C, Makefile, pkg-config, etc.
But seems like both are used: https://github.com/hoehermann/purple-signald/blob/v0.11.0/Makefile#L26 So I'll update to provide both?

The share doesn't work within nix for pidgin anyway, see: #8999. So not too important for me right now.

Reply:
Not configuring `PIDGIN_DATADIR` will result in:
```
install: cannot create regular file '/nix/store/maf4a5lyaijfxbzix7icha98019wbmkl-pidgin-2.14.10/share/pixmaps/pidgin/protocols/16/signal.png': Permission denied
```

Reply:
Oh right, DATADIR, not PLUGINDIR, of course. It should be fine to set both PIDGIN and PURPLE, in that case!